packages/trip-form/src/index.js

```diff
@@ -16,6 +17,7 @@ export {
   ModeButton,
   ModeSelector,
   SettingsSelectorPanel,
-  Styled,
-  SubmodeSelector
+  SubmodeSelector,
+  TripOptions,
+  Styled
```

Comment:
When I used this `Styled` export in the trimet.org app it works fine, but in our jest tests it comes through as `undefined`.  Haven't gotten to the bottom of why that is, but seeing some similar cases where its a circular dependencies (though I don't see that in this package).  Other posts are related to common-js config with babel, but I don't know why jest would be using that version of the code

So there may not be much for you to act on here, but wanted to at least document it and see if you had any ideas

Reply:
So I have more info on this issue.  I've confirmed that our Jest tests are using the CommonJS build from the `lib` folder, this appears to be the default setup as Jest's support for esm builds is [still experimental](https://jestjs.io/docs/ecmascript-modules).

I'm using the latest from the branch this PR is based on which I released to a trimet npm repo and when I looked at the index file in `lib` for the `trip-form` package most of the exports were there, but indeed `Styled` was not (even though it was there in `src/index`).  I have this branch checked locally so I tried deleting the lib folder for trip-form and rebuilding it with `yarn prepublish`, but only a fraction of the files that were there before got regenerated (see the image below).  So this does appear to be an issue with the CommonJs build, I just not sure if I'm doing something wrong in the process of generating it or if the build is broken (I'm on working on a Windows 10 machine btw)

![otp-ui-incomplete-common-js-build](https://user-images.githubusercontent.com/1002875/135346652-9f32d036-f73a-4ce3-a2b0-6b5c25a13f60.PNG)

Reply:
This is fixed in the latest alpha.  Is there anything else I need to do to generated the CommonJS build in addition to running `yarn prepublish`?